### PR TITLE
Let keepalived open a raw socket

### PR DESCRIPTION
**Pull request by mscherer**
See https://bugzilla.redhat.com/show_bug.cgi?id=1891895

### CODE REVIEW

File: keepalived.te [hunk 1]
```diff
@@ -43,6 +43,7 @@ allow keepalived_t self:netlink_connector_socket create_socket_perms;
 allow keepalived_t self:netlink_route_socket nlmsg_write;
 allow keepalived_t self:packet_socket create_socket_perms;
 allow keepalived_t self:rawip_socket create_socket_perms;
+corenet_raw_bind_generic_node(keepalived_t)
```
Comment (zpytela) [inline keepalived.te]: @mscherer could you move the line to the other corenet_ calls?

**Review by zpytela — CHANGES_REQUESTED** [review]
I think it should rather be binding than opening.

File: keepalived.te (hunk 1 above)
Comment (mscherer) [reply keepalived.te]: yup

**Comment by zpytela** [comment]
I tried to look up rules for ordering within corenet interfaces block usage, without success though. It seems to me the order is not alphabetical as it is in most of the cases, but rather all-generic-particular. Having said that, I'd rather move it at the block beginning.

**Review by zpytela — APPROVED** [review]

**Comment by zpytela** [comment]
@mscherer, thanks for the excellent cooperation.

**Comment by zpytela** [comment]
Cherry-picked for rawhide, too: https://github.com/fedora-selinux/selinux-policy-contrib/pull/352

**Pull request merged**